src/VSIX/NpgsqlProviderObjectFactory.cs

```diff
@@ -26,6 +26,8 @@ public override object CreateObject(Type objType)
                 return new DataObjectSupport($"{GetType().Namespace}.NpgsqlDataObjectSupport", Assembly.GetExecutingAssembly());
             if (objType == typeof(IVsDataViewSupport))
                 return new DataViewSupport($"{GetType().Namespace}.NpgsqlDataViewSupport", Assembly.GetExecutingAssembly());
+            if (objType == typeof(IVsDataConnectionEquivalencyComparer))
```

Comment:
Ok, I'll add to registration. [IVsDataConnectionEquivalencyComparer Interface](https://msdn.microsoft.com/en-us/library/microsoft.visualstudio.data.services.supportentities.ivsdataconnectionequivalencycomparer(v=vs.120).aspx) `{FE690B14-48F7-4687-81B1-44A111D2E919}`